### PR TITLE
docs(README.md): update min version of node to be 18

### PR DESCRIPTION
**Pull request by padamstx**
This project's package.json has specified version 18 as the min node version for a while now.  This commit just updates the README.md to reflect that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ This project contains core functionality required by Node.js code generated by t
 ```
 
 ## Prerequisites
-- Node.js version 14 or newer
-- Note that v14 is technically *supported* but not officially *guaranteed* as we only test on v16 and higher
+- Node.js version 18 or newer
+- Note that v18 is technically *supported* but not officially *guaranteed* as we only test on v16 and higher
```
Comment (dpopp07) [inline README.md:17]: We can remove this line - our minimum version and our build have been aligned.

**Review by dpopp07 — CHANGES_REQUESTED** [review]
Good catch! One small change to make though

**Review by dpopp07 — APPROVED** [review]
Looks good! 👍

File: README.md (hunk 1 above)
Comment (padamstx) [reply README.md]: done

**Pull request merged**